src/java.xml/share/classes/javax/xml/XMLConstants.java

```diff
@@ -28,6 +28,11 @@
 /**
  * <p>Utility class to contain basic XML values as constants.
  *
+ * @spec https://www.w3.org/TR/xml Extensible Markup Language (XML) 1.0 (Fifth Edition)
+ * @spec https://www.w3.org/TR/xml11 Extensible Markup Language (XML) 1.1 (Second Edition)
+ * @spec https://www.w3.org/TR/REC-xml-names Namespaces in XML 1.0 (Third Edition)
+ * @spec https://www.w3.org/TR/xml-names11 Namespaces in XML 1.1 (Second Edition)
+ * @spec https://www.w3.org/TR/xmlschema-1 XML Schema Part 1: Structures Second Edition
```

Comment:
Hi Jon,

I would agree with what Alan said earlier that the @see ref can be dropped. This particular class (XMLConstants.java [1]) is a good example for that argument: in the resulting javadoc, 5 specs were listed in the "External Specifications" section, 6 in "See Also:", and then they were listed again for each field. That's a lot of duplicates. Adding to the confusion was that the @spec and @see were not always the same, e.g. @spec XML 1.0.
points to the fifth edition while @see second.

A minor comment is that the '@spec's were rendered in one line while the @see refs a list. I would see the later is easier to read.

[1] http://cr.openjdk.java.net/~jjg/8296546/api.00/java.xml/javax/xml/XMLConstants.html

Reply:
The presentation of lists of `@spec` tags was fixed separately, and is incorporated into the latest docs that demo this work.   The same algorithm is now used for both `@see` and `@spec` tags ... if the links are short and do not contain commas, they will be displayed as an inline list; otherwise, they will be displayed in a bulleted list.